### PR TITLE
Adds transport support.

### LINKED ISSUE #1: Create HTTP transport

### PR DESCRIPTION
**Pull request by jcchavezs**
Ping @basvanbeek @marc-gr @adriancole @gchaincl

Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,29 @@
 
 [![Build Status](https://travis-ci.org/jcchavezs/zipkin-agent.svg?branch=master)](https://travis-ci.org/jcchavezs/zipkin-agent)
 
-Agent for Zipkin V2
+This is an agent that should run in the same host of the application in order to reduce the load
```
Comment (codefromthecrypt) [inline README.md]: reducing load is one thing, and not even necessarily on zipkin server (as spans end up in the server anyway). For example, some use agents to reduce load on kafka (which still ends up in zipkin, quite possibly with a similar load).

I'd suggest that running an agent reduces the load on the application, centralizes reporting functions such as bundling, letting restricted environments like php offload etc..

So, this indeed can reduce the load of zipkin, but it is very indirect.. it only reduces the load when the application is being very inefficient (like sending one span at a time)

File: README.md [hunk 2]
```diff
@@ -2,4 +2,29 @@
...
+go get github.com/jcchavezs/zipkin-agent
+go install github.com/jcchavezs/zipkin-agent/cmd/zipkin-agent
+```
+
+## Binary distribution
+Each release provides pre-built binaries for different architectures, you can download them here:
+https://github.com/jcchavezs/zipkin-agent/releases/latest
+
+## Getting started
+
```
Comment (codefromthecrypt) [inline README.md:30]: make a note how to download as not everyone are go. mention any dependencies or setup needed.

File: README.md [hunk 3]
```diff
@@ -2,4 +2,29 @@
...
+You can run the agent by:
+```bash
+zipkin-agent
+```
+
+the default transport is the standard stdout but you can also define the transport:
```
Comment (codefromthecrypt) [inline README.md]: stdout isn't a usable default. make someone choose or default to http (many tracing libraries default to http on localhost zipkin)

**Review by codefromthecrypt — COMMENTED** [review]
thx lookin good!

File: README.md (hunk 2 above)
Comment (jcchavezs) [reply README.md:30]: Good point. I will change it.

**Pull request merged**